octocrate/Cargo.toml

```diff
@@ -66,42 +67,42 @@ dotenv = { workspace = true }
 [[example]]
 name = "commits"
 path = "examples/commits.rs"
-required-features = ["repos"]
+required-features = ["integration", "repos"]
```

Comment:
It seems that `integration` is not listed in the features list, and the example does not require this feature